### PR TITLE
Re-enable H264 by default, set custom video preset, remove bc1 ICE st…

### PR DESCRIPTION
**Pull request by zackm0571**
…un default setting, disable hardware accelerated decoder for H.264.

### CODE REVIEW

File: Classes/LinphoneCoreSettingsStore.m [hunk 1]
```diff
@@ -112,10 +112,6 @@ - (void)transformCodecsToKeys:(const MSList *)codecs {
             if(![[[defaults dictionaryRepresentation] allKeys] containsObject:pref]){
                 [defaults setBool:TRUE forKey:pref];
                 [defaults synchronize];
-                if ([pref isEqualToString:@"h264_preference"]) {
-                    [defaults setBool:FALSE forKey:pref];
-                    [defaults synchronize];
-                }
             }
```
Comment (zackm0571) [inline Classes/LinphoneCoreSettingsStore.m:115]: Re-add H.264 to default codecs

File: Classes/LinphoneCoreSettingsStore.m [hunk 2]
```diff
@@ -282,14 +278,17 @@ - (void)transformLinphoneCoreToKeys {
...
         }
-		[self setCString:preset ? preset : "high-fps" forKey:@"video_preset_preference"];
+		[self setCString:preset ? preset : "custom" forKey:@"video_preset_preference"];
         MSVideoSize vsize;
         
         linphone_core_set_adaptive_rate_algorithm(lc, "Stateful");
         NSUserDefaults *defaults = [NSUserDefaults standardUserDefaults];
+        if(![[[defaults dictionaryRepresentation] allKeys] containsObject:@"video_preferred_size_preference"]){
+            [defaults setObject:@"vga" forKey:@"video_preferred_size_preference"];
+        }
```
Comment (zackm0571) [inline Classes/LinphoneCoreSettingsStore.m:291]: Set default video size to vga

File: Classes/LinphoneCoreSettingsStore.m [hunk 3]
```diff
@@ -282,14 +278,17 @@ - (void)transformLinphoneCoreToKeys {
 
 		const char *preset = linphone_core_get_video_preset(lc);
         if(!preset){
-            preset = "high-fps";
+            preset = "custom";
```
Comment (zackm0571) [inline Classes/LinphoneCoreSettingsStore.m:281]: Set video preset default to custom

File: Classes/LinphoneCoreSettingsStore.m [hunk 4]
```diff
@@ -282,14 +278,17 @@ - (void)transformLinphoneCoreToKeys {
...
             linphone_core_set_video_preset(lc, preset);
         }
-		[self setCString:preset ? preset : "high-fps" forKey:@"video_preset_preference"];
+		[self setCString:preset ? preset : "custom" forKey:@"video_preset_preference"];
         MSVideoSize vsize;
```
Comment (zackm0571) [inline Classes/LinphoneCoreSettingsStore.m:285]: Default video preset to custom

File: Classes/LinphoneManager.m [hunk 5]
```diff
@@ -1491,7 +1491,7 @@ - (void)createLinphoneCore {
 	libmswebrtc_init(f);
   
 	linphone_core_reload_ms_plugins(theLinphoneCore, NULL);
-    configH264HardwareAcell(true, true);
+    configH264HardwareAcell(true, false);
```
Comment (zackm0571) [inline Classes/LinphoneManager.m]: Disable hardware accelerated decoder. This decoder has proven to be unstable during tests and can produce completely gray iframes.

File: Classes/WizardViewController.m [hunk 6]
```diff
@@ -603,15 +603,15 @@ - (BOOL)addProxyConfig:(NSString*)username
...
-        MS_VIDEO_SIZE_ASSIGN(vsize, CIF);
+        MS_VIDEO_SIZE_ASSIGN(vsize, VGA);
         linphone_core_set_preferred_video_size([LinphoneManager getLc], vsize);
-        linphone_core_set_download_bandwidth([LinphoneManager getLc], 1000);
-        linphone_core_set_upload_bandwidth([LinphoneManager getLc], 1000);
+        linphone_core_set_download_bandwidth([LinphoneManager getLc], 1500);
+        linphone_core_set_upload_bandwidth([LinphoneManager getLc], 1500);
         
-        [[NSUserDefaults standardUserDefaults] setObject:@"Implicit" forKey:@"rtcp_feedback_pref"];
+        [[NSUserDefaults standardUserDefaults] setObject:@"Explicit" forKey:@"rtcp_feedback_pref"];
```
Comment (zackm0571) [inline Classes/WizardViewController.m:614]: Set RTCP rb to explicit by default to assist Sylvain with testing

**Comment by ladyhawk** [comment]
+1

**Pull request merged**